packages/javascript/FlatfileJavascript.ts

```diff
@@ -1,6 +1,7 @@
 import api from '@flatfile/api'
 import { Browser, FlatfileEvent, FlatfileListener } from '@flatfile/listener'
 
+import { Environment, Guest, Space, User, Workbook } from '@flatfile/api/api'
```

Comment:
Not a huge fan of this syntax, that folder may move inside the API package....
Can we do something like:
```ts
import Flatfile from '@flatfile/api'
/// then
Flatfile.Environment, Flatfile.Guest
```

Reply:
Oh yeah - that's terrible - I think my autocomplete did that - will definitely address